sdk/textanalytics/Azure.AI.TextAnalytics/src/DetectedLanguage.cs

```diff
@@ -1,24 +1,55 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text.Json;
+using Azure.Core;
 
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
     /// A prediction of the language in which a document is written in.
     /// </summary>
-    public readonly struct DetectedLanguage
+    [CodeGenModel("DetectedLanguage")]
```

Comment:
Although same name, because I am changing the namespace, I have to use `CodeGenModel`